packages/server-ct/src/server-ct.ts

```diff
@@ -1,28 +1,31 @@
-const _ = require('lodash')
-const http = require('http')
-const debug = require('debug')('cypress:server-ct:server')
-const express = require('express')
-const Bluebird = require('bluebird')
-
-const allowDestroy = require('@packages/server/lib/util/server_destroy')
-const templateEngine = require('@packages/server/lib/template_engine')
-const Socket = require('./socket-ct')
+import http from 'http'
+import { promisify } from 'util'
+import _debug from 'debug'
```

Comment:
I've been preferring `import debug as debugFn from 'debug'` inside of typescript -- but that's not a hard and fast convention

Reply:
I dont think `as` is valid for a default export